submarine-workbench/interpreter/spark-interpreter/src/test/resources/hive-site.xml

```diff
@@ -0,0 +1,7 @@
+<configuration>
```

Comment:
When will hive-site.xml be loaded?

Reply:
the SparkSQL will need hive-site to create a table.